pkg/cmd/roachtest/canary.go

```diff
@@ -143,7 +143,7 @@ func repeatRunWithBuffer(
 		var result []byte
 		result, lastError = c.RunWithBuffer(ctx, l, node, args...)
 		if lastError != nil {
-			c.l.Printf("error - retrying: %s", lastError)
+			c.l.Printf("error - retrying: %s\n%s", lastError, string(result))
```

Comment:
actually wouldn't it be more helpful if you printed the last `string(result)` in the `fmt.Errorf` below? Then the test harness will give you the output as a reason for the failing test and you're less likely to have to fish it from the logs.

Reply:
Good point. I think we should be printing out the output in both places.